ontology/yaml/resources/HVAC/entity_types/PMP.yaml

```diff
@@ -33,6 +33,13 @@ PMP_SS_VSC:
   - SS
   - VSC
 
+PMP_CWDPSS:
```

Comment:
This is not necessary. it is unnecessary to call out that it is for condensate, it is simply a pump. this should be PMP_SS, which already exists.